docs/data/material/components/stack/ZeroWidthStack.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Stack from '@mui/material/Stack';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+const Item = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+  ...theme.typography.body2,
+  padding: theme.spacing(1),
+  textAlign: 'center',
+  color: theme.palette.text.secondary,
+}));
+
+const message = `Truncation should be conditionally applicable on this long line of text
+ as this is a much longer line than what the container can support. `;
+
+export default function ZeroWidthStack() {
+  return (
+    <Box sx={{ flexGrow: 1, overflow: 'hidden', px: 3 }}>
+      <Item
+        sx={{
+          my: 1,
+          mx: 'auto',
+          p: 2,
+        }}
+      >
+        <Stack spacing={2} direction="row" alignItems="center">
+          <Avatar>W</Avatar>
+          <Typography noWrap>{message}</Typography>
+        </Stack>
+      </Item>
+      <Item
+        sx={{
+          my: 1,
+          mx: 'auto',
+          p: 2,
+        }}
+      >
+        <Stack spacing={2} direction="row" alignItems="center">
+          <Stack>
+            <Avatar>W</Avatar>
+          </Stack>
```

Comment:
Is this a typo?

```suggestion
          <Avatar>W</Avatar>
```

it looks wrong here too https://mui.com/joy-ui/react-stack/#white-space-nowrap in prod.

Reply:
I think so but let's fix it in another PR. It is not related to the `useFlexGap` prop.